assets/js/components/global/filters.js

```diff
@@ -0,0 +1,8 @@
+// site-wide navigation filters
```

Comment:
the `components` folder is for vue components, this isn't one.

move it to `assets/js/globals.js` -- we might export more globals later anyway. don't forget to update everywhere where it's imported.